apps/admin-gui/src/app/shared/components/dialogs/attribute-export-dialog/attribute-export-dialog.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { RemoveGroupResourceDialogData } from '../remove-group-resource-dialog/remove-group-resource-dialog.component';
```

Comment:
unused import

Reply:
This dialog has been removed completely.